### PR TITLE
devsim: Integrate vku library

### PR DESCRIPTION
**Pull request by jeremyk-lunarg**
Integrated new `vku` library to read layer settings.

### CODE REVIEW

File: build-android/jni/Android.mk [hunk 1]
```diff
@@ -18,6 +18,7 @@ LOCAL_PATH := $(call my-dir)
 SRC_DIR := ../..
 LAYER_DIR := ../generated
 THIRD_PARTY := ../third_party
+VKU_DIR := ../third_party
```
Comment (lunarpapillo) [inline build-android/jni/Android.mk:21]: this looks like it is not used?

Perhaps it should be `$(SRC_DIR)/vku` and used in both `LOCAL_SRC_FILES` and `LOCAL_C_INCLUDES` below.

**Review by lunarpapillo — CHANGES_REQUESTED** [review]
One minor change, then consider it approved by me.

**Review by christophe-lunarg — APPROVED** [review]

**Comment by christophe-lunarg** [comment]
This PR should be merge for SDK testing.

File: build-android/jni/Android.mk (hunk 1 above)
Comment (christophe-lunarg) [reply build-android/jni/Android.mk]: Good catch, I removed it.

**Pull request merged**